api/.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+AllCops:
+  Exclude:
+    - db/schema.rb
+Style/HashEachMethods:
+  Enabled: true
+Style/HashTransformKeys:
+  Enabled: true
+Style/HashTransformValues:
+  Enabled: true
+Layout/LineLength:
+  Max: 120
+  Exclude:
+    - config/initializers/**
+    - spec/models/**
```

Comment:
I would disagree with this exclusion. This rule make code more readable. In very-very rare cases you can disable rule for a line.